apps/workbench/config/application.yml.example

```diff
@@ -24,6 +24,9 @@ production:
   secret_token: ~
 
   # You probably also want to point to your API server.
-  arvados_login_base: https://arvados.local:3030/login
-  arvados_v1_base: https://arvados.local:3030/arvados/v1
+  arvados_login_base: https://uuid_prefix.your.domain/login
+  arvados_v1_base: https://uuid_prefix.your.domain/arvados/v1
   arvados_insecure_https: false
+
+  # You might also want to name your workbench
+  site_name: ~
```

Comment:
I agree it's more convenient at setup time to have everything right there in the editable config file. The problem is that your customized copy of the example file isn't touched during upgrades, so after a few upgrades the convenience bites back: the comments (including "default is X") no longer match what the software actually does. "Look in the actual defaults/docs file for more info" is less convenient during the initial setup, but at least it doesn't start misleading you later.

`site_name` isn't essential so it doesn't belong in the minimal file. "Mostly interesting" varies from one user to another. How about improving the comments in the example file to encourage people to look in `application.default.yml` and decide which options are most interesting to them?

We could also point out in the example file that if you prefer a big editable template (and don't mind the comments/defaults going out of sync when you upgrade the software) you can simply use a copy of `application.default.yml` as the starting point for your `application.yml`. AFAIK that should work fine.